resources/views/asv/index.blade.php

```diff
@@ -0,0 +1,62 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="featuredLoadoutsContainer pb-20 pt-3">
+        <div class="flex flex-row lg:justify-start flex-wrap pl-10">                        
+            <x-equipment-component 
+                :gun="$gun" 
+                :modMatrix="$modMatrix" 
+                :overclock="$overclock"
+                class="asvCol">
+            </x-equipment-component>
+
+            <div class="md:pl-20 lg:pl-20 asvCol">
+                <h3>Burst DPS</h3>
+                <p>Ideal: {{$buildMetrics['ideal_burst_dps']}}</p>
```

Comment:
Should we be rounding any of this data?

Reply:
This is a question for @phg49389

Reply:
I choose to round it to 4 decimals in my program, but you can round it however you think looks best for the website. The values I give in that table are as precise as I can give you, which lets you choose how they get displayed.